src/test/resources/load_config_test.properties

```diff
@@ -2,15 +2,15 @@
 # to simulate a load test with 5 concurrent users then you need to enter 5 as the value for this property. A high end
 # machine will be able to spawn more number of threads. To keep the consistent(or nearly consistent) gap between the
 # threads, adjust this number with 'ramp.up.period.in.seconds' and the actual response time of the API end point.
```

Comment:
Update this `property` how this affects multiple load

Reply:
Updated this file as well as wiki page.

Reply:
Worth creating a WikiPage for this multi-load feature

Reply:
#91 has been raised for this.